### DISCUSSION TITLE
How to sync changes in lib\site-packages\notifications\base\

### DISCUSSION THREAD

**Discussion by DamyanBG** [python-poetry/poetry#6782] [Q&A] [answered]
Hello,

I have some deprecation warnings related to distlib. I opened the files which have this package and changed it with packaging. Okay, but how to sync my changes ?

**Comment by neersighted** [comment]
It's unclear what your exact issue is, but if you are referring to the deprecation of distutils, some of the warnings are spurious and related to pip internals -- you may be encountering those. More details are needed to even understand your issue, but you should definitely **not** be editing files in site-packages.

**Comment by DamyanBG** [comment]
First I want to thank you for the fast reply :)

Yes, my issue was 100% clear, but you understood it and helped me.

I have deprecation warnings because if distutils in package notification which checks the version of django. I fixed that in my wrong way, but still I have to find better solution, to make the fix and to sync my poetry virtual environment with the other teammates.

**Comment by neersighted** [comment, marked as answer]
If you're referring to code like this:

https://github.com/django-notifications/django-notifications/blob/e8d2a251c227ea0e48cf77fd52a91a6cb6882778/notifications/urls.py#L1-L3

In django-notifications, I'm afraid you'll have to get it it fixed upstream, or fork it and depend on it as a Git-based dependency.

**Comment by DamyanBG** [comment]
Exactly this code makes the trouble.
